docs/madr/decisions/036-internal-listeners.md

```diff
@@ -0,0 +1,15 @@
+# Internal listeners naming convention
+* status: accepted
+
+## Context and Problem statement
+At the moment we don't clearly distinguish between listeners used for service to service communication
+and internal listeners, like Prometheus listeners.
+
+## Decision Outcome
+
+We will add prefix `_` to internal listeners.
+
+## Solution
+At the moments internal listeners names start with `kuma` prefix, problem is that service can also have this prefix.
```

Comment:
Could we add a note about a migration? Users might have dashboards that rely on cluster names and that would break it after an upgrade. Should we add a configuration option to turn off this behavior?

Reply:
added section about updating UPGRADE.md after changing naming of already existing resources